packages/api/db/migrations/20200712133735_create_totaltests_column.ts

```diff
@@ -0,0 +1,16 @@
+import Knex from "knex";
+
+export const up = async (knex: Knex) => {
+    return knex.schema.table("users", table => {
```

Comment:
You can always just leave out return and the braces, see how I do it

Reply:
Well you don't always do it.